playbooks/roles/appsembler_reporting/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: make logs directory for appsembler_reporting
+  file:
+    path: "{{ appsembler_reporting_log_dir }}"
+    state: "directory"
+    owner: "{{ edxapp_user }}"
+    group: syslog
+    mode: 0755
+
+- name: Create the reporting scripts directory
+  file:
+    path: "{{ appsembler_reporting_scripts_dir }}"
+    state: directory
+    owner: edxapp
+    group: edxapp    
+
+- name: Create the reporting scripts
+  copy:
+    content: >
+      source {{ edxapp_app_dir}}/edxapp_env;
+      /edx/bin/python.edxapp /edx/bin/manage.edxapp lms --settings=aws_appsembler  \
+        generate_appsembler_report {{ item.key }} --no-delay  \
+        2>&1 | tee -a {{ appsembler_reporting_log_dir }}/{{ item.key }}_report.log
+    dest: "{{ appsembler_reporting_scripts_dir }}/{{ item.key }}_report.sh"
+    owner: edxapp
+    group: edxapp
+    mode: 774
```

Comment:
`774` doesn't make sense here. You could do it `770` and it would better. As you're giving `4` (read) to `others` and but not `1` (execute) so it's useless.